src/api/requests/downloadValidationReport.ts

```diff
@@ -0,0 +1,44 @@
+import type { SblAuthProperties } from 'api/useSblAuth';
+import axios from 'axios';
+import type { FilingPeriodType } from 'types/filingTypes';
+
+export interface DownloadValidationReportProperties {
+  auth: SblAuthProperties;
+  lei: string;
+  filingPeriod: FilingPeriodType;
+}
+
+export const downloadValidationReport = async ({
+  auth,
+  lei,
+  filingPeriod,
+}: DownloadValidationReportProperties): Promise<void> => {
+  try {
+    await axios({
+      headers: { Authorization: `Bearer ${auth.user?.access_token}` },
+      url: `/v1/filing/institutions/${lei}/filings/${filingPeriod}/submissions/latest/report`,
+      method: 'GET',
+      responseType: 'blob',
+    }).then(response => {
+      const url = window.URL.createObjectURL(
+        new Blob([response.data], { type: 'text/csv;charset=utf-8' }),
+      );
+      const link = document.createElement('a');
+      link.href = url;
+      link.setAttribute(
+        'download',
+        `${lei}-${filingPeriod}-validation_report.csv`,
```

Comment:
The one item I would add is the Submission ID.  If a user submits multiple times, it would be helpful to be able to differentiate which iteration a validation report aligns to.